__tests__/app.test.js

```diff
@@ -127,6 +127,35 @@ describe('GET requests', () => {
             })
         });
     })
+    describe('GET /api/reviews', () => {
+        it('should responds with 200 and an array of review objects sorted in date order', () => {
+            return request(app)
+            .get('/api/reviews')
+            .expect(200)
+            .then(({body}) => {
+                expect(body.reviews).toBeSortedBy('created_at', {descending:true})
```

Comment:
I'd recommend you also have another expect statement here which checks the structure of the object of the articles... i.e. does /api/reviews return an array of objects  which include the keys... owner/title/review_id and so on with which expect.any(x) values. You want confidence that this test has returned all the articles for you.

Maybe also an expect statement in addition which checks how many (length) articles are returned